typescript/cli/package.json

```diff
@@ -11,6 +11,7 @@
     "@inquirer/core": "9.0.10",
     "@inquirer/figures": "1.0.5",
     "@inquirer/prompts": "^3.0.0",
+    "@inquirer/search": "^3.0.1",
```

Comment:
not for this PR but could you create a separate issue to investigate and bump up the inquirer packages across the board to more recent versions? 

i ask because prompts/figures don't look upto date, and v3 of search seems to be wanting v10+ of core (we only specify v9 here)

Reply:
will do sir! thanks for pointing it out